### DISCUSSION TITLE
pivpn-tap-web-ui

### DISCUSSION THREAD

**Discussion by karabelnikov** [bnhf/openvpn-admin-plus#38] [Q&A] [answered]
**@bnhf**

Scott, the project has long gone to another level and is working with **TUN**. Maybe you should remove the abbreviation **TAP** from the name of the branch and the project? I think we can stop at the name **`pivpn-web-ui`**.

**Comment by bnhf** [comment]
@karabelnikov 

I like this idea, though I'd suggest we only change the name of the repository, not the package name. Lots of changes if we modify the name in the code.

pivpn-web-ui is probably the most descriptive, though openvpn-admin or openvpn-admin-plus would probably get more hits in search engines.

As soon as we stop making so many changes, I'm going to do a write-up on my blog, and perhaps we can do some other types of promotion. Having an improved name before that would make sense.

**Comment by karabelnikov** [comment]
**@bnhf** 

I agree, it's better to use openvpn-admin or openvpn-admin-plus.

**Comment by bnhf** [comment, marked as answer]
@karabelnikov 

Name change to openvpn-admin-plus has been done. We'll see what happens! I also posted in Reddit about the name change. In addition, there's another person that has been doing some updating to this project. I reached out to him to see if he's interested in collaboration, rather than duplication. Check out his project when you have a moment to see if there's anything we should incorporate into this one. Or better yet, hopefully we'll see him here in the future. :-)

https://github.com/d3vilh/raspberry-gateway

**Comment by karabelnikov** [comment]
Hi, Scott! Wthe past holidays to you! :) It's great that the project got a new name, I think this is the right way. We have implemented a lot in our new project. It's a great job. It will, we will improve it again and again.

I looked at another person's project using the link you wrote. To be honest, I didn't see anything interesting in his project. In fact, his project remained at the same level as it was, outdated. Yes, he added a few new features that are also in our project with a better implementation. The only thing we can add is a button to restart the openvpn server, we need to think about this implementation.

It would be nice if he joined our project and suggested some new ideas and functions that could be implemented. I support it!
